robot/EDA/resources/CourseConnectionsSettingsPageObject.py

```diff
@@ -20,3 +21,50 @@ def _is_current_page(self):
             locator_tab,
             error=f"Course Connections tab with locator '{locator_tab}' is not available on the page"
         )
+
+    def set_enable_course_connections(self):
+        """ Set the checkbox for 'Enable Course Connections' field """
+        locator = eda_lex_locators["eda_settings_cc"]["enable_cc_checkbox"]
+        self.selenium.wait_until_page_contains_element(locator)
+        self.selenium.click_element(locator)
+
+    def update_enable_cc_to_default(self):
+        """ Updating the `Enable Course Connections` checkbox to default value (false)"""
```

Comment:
looks like first u r checking and updating only if not correct. Maybe u can adjust ur documentation to reflect that so that its clear if anyone else wants to use

Reply:
Updated